src/project/internal/notationproject.cpp

```diff
@@ -337,6 +337,31 @@ mu::Ret NotationProject::doSave(bool generateBackup)
     return make_ret(Ret::Code::Ok);
 }
 
+mu::Ret NotationProject::makeCurrentFileAsBackup()
+{
+    io::path filePath = m_engravingProject->path();
+    if (io::suffix(filePath) != engraving::MSCZ) {
+        LOGW() << "backup alowed only for MSCZ, current: " << filePath;
```

Comment:
Typo: `allowed`. And probably `currently`

Reply:
Thanks, fixed!